nightwatch/commands/endSauceAnnotation.js

```diff
@@ -42,7 +42,7 @@ module.exports = class CustomCommand {
       this.api.currentTest.results.testcases[jobName].failed +
         this.api.currentTest.results.testcases[jobName].errors ===
       0;
-    this.api.execute(`sauce:job-result=${passed}`);
-    this.api.execute(`sauce:job-name=${moduleName + jobName}`);
+    await this.api.execute(`sauce:job-result=${passed}`);
+    await this.api.execute(`sauce:job-name=${moduleName + jobName}`);
```

Comment:
Turns out this.api.execute is `Awaitable`. Decided the make the command `async` so the `Awaitable` makes sense to have in the type